packages/pn-pa-webapp/src/pages/components/Notifications/FilterNotifications.tsx

```diff
@@ -84,8 +80,8 @@ const FilterNotifications = forwardRef(({showFilters}: Props, ref) => {
       const currentFilters = {
         startDate: values.startDate.toISOString(),
         endDate: values.endDate.toISOString(),
-        recipientId: values.recipientId || undefined,
-        iunMatch: values.iunMatch || undefined,
+        recipientId: getAorB(values.recipientId, undefined),
+        iunMatch: getAorB(values.iunMatch, undefined),
         status: values.status === 'All' ? undefined : values.status,
```

Comment:
proviamo a unificare il tipo tra currentFilter e initialValues (o solo string o in entrambi i casi string | undefined)
in modo da usare univocamente la getValue di sopra

Reply:
Ciao, Andrea. Stai guardando un vecchio commit, però.

Reply:
No no ho guardato quello nuovo